02_Download_Requirement_Files.bat

```diff
@@ -82,5 +85,4 @@ echo ===
 echo Download Error
 echo ===
 rundll32 user32.dll,MessageBeep
-pause
-
+pause
```

Comment:
It is abnormal for differences to occur in this location .
Please return to a state where there are no differences .
(When there are strange differences like this , I lose the desire to review .)